pkg/policy/api/rule.go

```diff
@@ -183,16 +183,13 @@ type ServiceSelector EndpointSelector
 
 // Service wraps around selectors for services
 type Service struct {
-	// K8sServiceSelector selects services by k8s labels. Not supported yet
-	K8sServiceSelector ServiceSelector `json:"k8sServiceSelector,omitempty"`
-	// K8sService selects service by name and namespace pair
-	K8sService K8sServiceNamespace `json:"k8sService,omitempty"`
```

Comment:
By removing `k8sService` json tag aren't you breaking backwards compatibility?

Reply:
I guess I am. Do you think it should be left there?

Reply:
>Do you think it should be left there?

Yes